kolibri/locale/mr/LC_MESSAGES/kolibri.plugins.learn.app-messages.json

```diff
@@ -1,6 +1,6 @@
 {
   "AllClassesPage.noClasses": "तुम्हाला कोणताही वर्ग नेमून दिलेला नाही",
-  "AnswerHistory.question": "प्रश्न { num }",
+  "AnswerHistory.question": "प्रश्न xxx  {num, number, integer}",
```

Comment:
@radinamatic this looks a little weird. is this "xxx" actually part of the localization?

Reply:
@jonboiser  this is my bad, I added some test strings to test on hot reload, fixing it :cry: